notebooks/Wczytanie_danych.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
Po co nam drugi plik wczytanie danych? Do czego ma on służyć? Czy to nie niepotrzebne zduplikowanie notatników?